app/shared/db/breakpoints.server.ts

```diff
@@ -28,12 +32,30 @@ export const create = async (
   treeId: Project["id"],
   breakpoints: Array<Breakpoint>
 ) => {
-  const data = {
+  await prisma.breakpoints.create({
+    data: {
+      treeId,
+    },
+  });
+
+  const all = await prisma.$transaction(
+    breakpoints.map((breakpoint: Breakpoint) =>
```

Comment:
might be a stupid question because I haven't used sqlite yet, but why  not put breakpoint values in the create call above?